content/reference/current/plugins/github_actions/index.md

```diff
@@ -0,0 +1,230 @@
+
+{{< img src="quickstart_run_workflow.png" alt="Run workflow menu" >}}
+
+## Configuration reference
+
+Several configuration options are available as Action inputs. This Action also provides several outputs which you can access in the following steps of your workflow.
+
+### Inputs
+
+Example:
+
+```yaml
+steps:
+  - uses: gatling/enterprise-action@v1
+    with:
+      gatling_enterprise_url: https://cloud.gatling.io
+      api_token: ${{ secrets.GATLING_ENTERPRISE_API_TOKEN }}
+      simulation_id: '00000000-0000-0000-0000-000000000000'
+      extra_system_properties: >
+        {
+          "sys_prop_1":"value 1",
+          "sys_prop_2":42,
+          "sys_prop_3":true
+        }
+      extra_environment_variables: >
+        {
+          "ENV_VAR_1":"value 1",
+          "ENV_VAR_2":42,
+          "ENV_VAR_3":true
+        }
+      fail_action_on_run_failure: true
+      wait_for_run_end: true
+```
+
+- `gatling_enterprise_url` {{< badge danger >}}required{{< /badge >}}: The URL for your Gatling Enterprise Self-Hosted instance (if not specified, it will point to Gatling Enterprise Cloud instead).
+
+- `api_token` {{< badge danger >}}required{{< /badge >}} (unless using an environment variable named `GATLING_ENTERPRISE_API_TOKEN` instead): The API token used by the Action to authenticate with Gatling Enterprise.
+
+- `simulation_id` {{< badge danger >}}required{{< /badge >}}: The ID of the simulation as configured on Gatling Enterprise.
+
+- `extra_system_properties` {{< badge info >}}optional{{< /badge >}}: Additional Java system properties, will be merged with the simulation's configured system properties. Must be formatted as a JSON object containing the desired key/value pairs. Values can be strings, numbers or booleans.
+
+- `extra_environment_variables` {{< badge info >}}optional{{< /badge >}}: Additional environment variables, will be merged with the simulation's configured environment variables. Must be formatted as a JSON object containing the desired key/value pairs. Values can be strings, numbers or booleans.
+
+- `fail_action_on_run_failure` {{< badge info >}}optional{{< /badge >}} (defaults to `true`): If `true`, the Action will fail if the simulation run ends in an error (including failed assertions). Note: if set to `false` and the simulation ends in an error, some of the outputs may be missing (e.g. there will be no assertion results if the simulation crashed before the end).
+
+- `wait_for_run_end` {{< badge info >}}optional{{< /badge >}} (defaults to `true`): If `true`, the Action will wait for the end of te simulation run on Gatling Enterprise before terminating. Note: if set to `false`, some of the outputs may be missing (there will be no status nor assertion results).
+
+### Outputs
+
+Example:
+
+```yaml
+steps:
+  - id: gatling-enterprise-action
+    uses: gatling/enterprise-action@v1
+    with:
+      gatling_enterprise_url: https://cloud.gatling.io
```

Comment:
```suggestion
      gatling_enterprise_url: http://your-gatling-self-hosted-instance-domain
```

Reply:
`https://cloud.gatling.io` is used in the examples for the other CI plugins (by the way, looking at the history for the Jenkins doc, the example used to be `https://demo-beta.gatling.io`, and before that `http://my-website:10542`). If we change that we should probably be consistent about it.

Either way, I think it's slightly clearer with something that looks more like a real example rather than a placeholder - could be something like `http://my-gatling-instance.my-domain.tld` or similar, if we no longer want to use the cloud domain as the example. But maybe that's just me. 🤷

Reply:
Guess we should modify the URL to be http://my-gatling-instance.my-domain.tld or similar everywhere.

Reply:
The default value is/should be `"https://cloud.gatling.io"`
But as an example, we want to show that they can put their instance.
So, yes, I'm in favor of a constant like `http://my-gatling-instance.my-domain.tld` everywhere.
Obviously, the "everywhere" part is not the subject of this PR.